### ISSUE TITLE
Rename "sprite folders" addon

### ISSUE THREAD

**Issue by WorldLanguages** [ScratchAddons/ScratchAddons#5669]
### Why this would be helpful

I've seen it's pretty common to use the folders addon to group costumes. The current name might be confusing.

### My suggestion

Rename the addon. My suggestion: "sprite and costume folders"

### Possible alternatives

We could also name it "sprite, costume and sound folders" but I don't know, might be too long.
It could also just be "folders" but that could be a bit ambiguous.

**Comment by Samq64** [comment]
I'm thinking "Editor folders" but I'm not usre how much better that is compared to just "Folders".

**Comment by DNin01** [comment]
If the plan is to rename it, though, I'd vote for "sprite/asset folders".

**Comment by BroJac5246** [comment]
> > I'm thinking "Editor folders" becuase I like short names, but I'm not sure how much better that is compared to just "Folders".
> 
> To me, with that summarization, the subject is a little unclear. (Folders of _what_ in the editor? Or even the editor itself?)
> 
> I'm fine keeping it the way it is right now - sprites are the most obvious thing it would be used/useful for, and we can also specify that the addon also lets you make folders of costumes and sounds (if I'm right).

I think it's good to make it clear that costumes and sounds can be put in folders. I agree with Samq64, and something like "Folders in the [project?] editor" would work if "Editor Folders" is deemed unclear.

**Comment by mxmou** [comment]
> If the plan is to rename it, though, I'd vote for "sprite/asset folders".

I don't think we currently use the word "asset" anywhere. Editor dark mode has a setting called "selected sprite/costume/sound background".

**Comment by lisa-wolfgang** [comment]
I think "editor folders" makes sense considering we also have "editor find bar" and "editor devtools".

**Comment by WorldLanguages** [comment]
> Might be a little ambiguous. To me, with that summarization, the subject is a little unclear. (Folders of _what_ in the editor? Or even the editor itself?)

Agreed. What would "editor folders" mean? Folders of what?

**Comment by WorldLanguages** [comment]
Why not just `Sprite and costume folders`? Even though sounds aren't explicitly mentioned, it is better to use "costume" than "asset". Do users even know what an asset is?

**Comment by BroJac5246** [comment]
> Why not just `Sprite and costume folders`? Even though sounds aren't explicitly mentioned, it is better to use "costume" than "asset". Do users even know what an asset is?

I would prefer `Sprite and asset folders`. And if any users don't know what assets are, their browser has a search bar for a reason :P

**Comment by DNin01** [comment]
> I think "editor folders" makes sense considering we also have "editor find bar" and "editor devtools".

That's true, but it still wouldn't be my first choice.

**Comment by penguinmoose** [comment]
I think `Sprite and asset folders` is ok. If users don't know what assets are, they will still know that the addon makes it possible to create sprite folders.

**Comment by DNin01** [comment]
> I think `Sprite and asset folders` is ok. If users don't know what assets are, they will still know that the addon makes it possible to create sprite folders.

We could also add "assets (costumes and sounds)" to the description to teach the user what "assets" means.

**Comment by WorldLanguages** [comment]
We don't use the word "asset" anywhere else in the addon strings

**Comment by BroJac5246** [comment]
> We don't use the word "asset" anywhere else in the addon strings

It's still the most correct term, though, so I think we should use it.

**Comment by mxmou** [comment]
> > We don't use the word "asset" anywhere else in the addon strings
> 
> It's still the most correct term, though, so I think we should use it.

We don't have to use the same words as Scratch uses internally - it's more important that users can understand what the word means. For example, we don't call the code area the "workspace" and custom blocks "procedures" even though those words are used frequently in Scratch's source code.
